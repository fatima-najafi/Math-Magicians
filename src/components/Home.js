import './Style.css';

function Home() {
  return (
    <div className="home">
      <h2>Welcome to our page</h2>

      <div className="content">
        <p className="text">
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo
          labore rerum sunt debitis vitae odio mollitia neque, accusamus rem
          optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem!
          Repellendus. Lorem ipsum dolor sit, amet consectetur adipisicing elit.
          Dolores quo labore rerum sunt debitis vitae odio mollitia neque,
          accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa
          ratione quidem! Repellendus.
        </p>

        <p className="text">
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo
          labore rerum sunt debitis vitae odio mollitia neque, accusamus rem
          optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem!
          Repellendus. Lorem ipsum dolor sit, amet consectetur adipisicing elit.
          Dolores quo labore rerum sunt debitis vitae odio mollitia neque,
          accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa
          ratione quidem! Repellendus.
        </p>
      </div>
    </div>
  );
}
export default Home;
