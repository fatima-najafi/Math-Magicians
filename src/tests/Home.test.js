import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

test('renders the home', () => {
  render(<Home />);
  const welcomeText = screen.getByText(/Welcome to our page/i);
  expect(welcomeText).toBeInTheDocument();
});

test('matches snapshot', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
