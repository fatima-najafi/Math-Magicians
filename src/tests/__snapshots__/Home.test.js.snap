// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="home"
  >
    <h2>
      Welcome to our page
    </h2>
    <div
      class="content"
    >
      <p
        class="text"
      >
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo labore rerum sunt debitis vitae odio mollitia neque, accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem! Repellendus. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo labore rerum sunt debitis vitae odio mollitia neque, accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem! Repellendus.
      </p>
      <p
        class="text"
      >
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo labore rerum sunt debitis vitae odio mollitia neque, accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem! Repellendus. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolores quo labore rerum sunt debitis vitae odio mollitia neque, accusamus rem optio saepe corporis ipsa necessitatibus eaque, culpa ratione quidem! Repellendus.
      </p>
    </div>
  </div>
</div>
`;
